lib/pages/feed/item_list.dart

```diff
@@ -0,0 +1,172 @@
+import 'package:flutter/widgets.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:intl/intl.dart';
+
+import '../../common/colors.dart';
+import '../../services/pixel_service.dart';
+import '../../models/pixel.dart';
+import '../../models/comment.dart';
+import '../../widgets/comment/comment_list.dart';
+
+const containerMargin = EdgeInsets.only(left: 10, right: 10, top: 13, bottom: 13);
+
+class ItemList extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+        height: 480,
+        margin: containerMargin,
+        color: Colors.detailBackground,
+        child: FutureBuilder(
+            future: PixelService.fetchPixel(),
+            builder: (BuildContext context, AsyncSnapshot<PixelModel> snapshot) {
+              if (snapshot.hasData) {
+                return Column(
+                    children: [
+                      _pixelDateTime(snapshot.data.createdAt),
+                      _picture(snapshot.data.url),
+                      _authorAndInsights(snapshot.data.author, likesCount: snapshot.data.likes, commentsCount: snapshot.data.comments.length),
+                      _comments(snapshot.data.comments)
+                    ]
+                );
+              } else if (snapshot.hasError) {
+                return Text("Somethis is wrong");
+              } else {
+                return Text("Loading");
+              }
+            }
+        )
+    );
+  }
+}
+
+Widget _pixelDateTime(DateTime dateTime) {
+  String dateFormat = DateFormat.yMMMMEEEEd().format(dateTime).toString();
+  return Row(
+      children: [
```

Comment:
Don’t use methods that returns Widgets to reuse code, this is a common anti pattern which prevents some performance routines from Flutter, instead build small StatelessWidgets

Reply:
Oh wow, you're completely right. I searched a bit after reading your comment and found [this](https://medium.com/flutter-community/splitting-widgets-to-methods-is-a-performance-antipattern-16aa3fb4026c). I might have to revisit my `comment list` widget later.

Reply:
I mean, I know the performance losses will probably not be a big issue for our app, but since a lot of the motivation on building this app is for learning, I think it is nice to avoid this anti pattern :)

Reply:
I got it! Changing it right now. You are right, the main proposal is to learning flutter.